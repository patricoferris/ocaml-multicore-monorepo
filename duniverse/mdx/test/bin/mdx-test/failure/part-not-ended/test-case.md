
```ocaml file=parts-begin-end.ml,part=toto
```
