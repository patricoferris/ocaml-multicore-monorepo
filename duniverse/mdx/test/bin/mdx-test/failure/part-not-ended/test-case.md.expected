File "test-case.md", lines 2-3: Error in the OCaml file include code block
In file ./parts-begin-end.ml, line 18: Part toto has no end.
